projects/plugins/jetpack/jetpack-user-content-tracking-redirects.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * User Content Tracking Redirects
+ *
+ * The purpose of this file is to track user generated link clicks on the emails and redirect them to the original URL.
+ * This is done by generating an iframe pointing to the track and redirect logic in .com.
+ *
+ * @package automattic/jetpack
+ */
+
+add_action( 'init', 'jetpack_user_content_tracking_redirect' );
+
+/**
+ * Generate the iframe to track and redirect the user generated link clicks.
+ */
+function jetpack_user_content_tracking_redirect() {
+	if ( ! isset( $_SERVER['QUERY_STRING'] ) ) {
+		return;
+	}
+
+	$query_params = sanitize_text_field( wp_unslash( $_SERVER['QUERY_STRING'] ) ); // sanitization
+	?>
+	<iframe id="trackingIframe" src='<?php echo esc_url( "https://wordpress.com/?$query_params" ); ?>'></iframe>
```

Comment:
I'm thinking it either should come from `subscribe.wordpress.com` (subscriptions and checkout forms come from here) or `public-api.wordpress.com` (existing email tracking links go through here). Once your corresponding WPCOM patch is ready we can discuss this.

Reply:
The patch is using `subscribe.wordpress.com` at the moment.